server/db/models/Stretch.js

```diff
@@ -39,4 +40,43 @@ const Stretch = db.define('stretch', {
   }
 })
 
+Stretch.getStretchesByCohort = async function(cohortId) {
+  const cohortStretches = await CohortStretch.findAll({
+    where: {
+      cohortId
+    }
+  })
+
+  const stretchesByCohort = []
+  for (let i = 0; i < cohortStretches.length; ++i) {
+    let stretch = await Stretch.findOne({
+      where: {
+        id: cohortStretches[i].stretchId
+      }
+    })
+    stretchesByCohort.push(stretch)
+  }
+  return stretchesByCohort
+}
+
+Stretch.getStretchesByCohortAndStatus = async function(cohortId, status) {
+  const cohortStretches = await CohortStretch.findAll({
```

Comment:
same refactoring as above. To get both in one function, can do following right at start of function:

let filterObject = {cohortId}
if (status) filterObject.status = status

Stretch.findAll({
include: [
{
model: CohortStretch,
where: filterObject
}
]
})
.then(students => res.json(students))
.catch(next)